### PR TITLE
Many changes

### PR DESCRIPTION
**Pull request by Houlistonm**
1. format indents for code blocks
1. added language to code blocks where reasonable
1. attempted code-copy button, had to back it out.
1. updated configs to account for changes (nats -> nats-tls)
1. fixed some typos.

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script src="/assets/scripts/copyCode.js"></script>
```
Comment (ameowlia) [inline _layouts/post.html]: Remove since not working

File: _includes/codeHeader.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="code-header">
```
Comment (ameowlia) [inline _includes/codeHeader.html]: Remove since not working

File: assets/css/main.scss [hunk 3]
```diff
@@ -17,3 +17,7 @@ h2 {
 .masthead__inner-wrap {
   max-width: 100%;
 }
+
+.copy-code-button {
```
Comment (ameowlia) [inline assets/css/main.scss]: Remove since not working

File: assets/scripts/copyCode.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// This assumes that you're using Rouge; if not, update the selector
```
Comment (ameowlia) [inline assets/scripts/copyCode.js]: Remove since not working

**Review by ameowlia — CHANGES_REQUESTED** [review]
Wow! Thank you so much for all of these improvements!

I tried for 30 minutes and couldn't get the copy code stuff working either.

* Can you remove it from this PR before I merge?
* Can you open a new PR with it and any helpful links that you were cribbing from? Then I can poke at it more when I have free time.

Thank you!

File: _config.yml [hunk 5]
```diff
@@ -288,3 +288,6 @@ defaults:
       comments: # true
       share: false
       related: false
+
+theme: jekyll-theme-midnight
```
Comment (ameowlia) [inline _config.yml:292]: Does this theme work for you? It doesn't change anything on my end.

Comment (ameowlia) [inline _config.yml]: Does this theme work for you? It doesn't change anything on my end. If it doesn't do anything, can you remove this line?

File: _config.yml [hunk 6]
```diff
@@ -288,3 +288,6 @@ defaults:
...
+baseurl: /cf-networking-onboarding-1
```
Comment (ameowlia) [inline _config.yml]: Are you able to run the site with this line? It breaks the site on my machine. If we wanted to update the base URL we would need to move all of the files and update lots of links I think. 

Can you remove it from this PR and if you want to introduce it, do it in a different PR with fixes for the other things mentioned above?

**Review by ameowlia — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 6 above)
Comment (Houlistonm) [reply _config.yml]: Done ...
Don't know where the theme came from, but the base url is required (my fork is named different for some reason).

**Comment by ameowlia** [comment]
TY so much for all of this work!
